### PR TITLE
Sync with community and update to latest release

### PR DESCRIPTION
**Pull request by juangon**
Had to add community io.thorntail:arquillian, as it isn't productized

### CODE REVIEW

File: greeting-service/.openshiftio/application.yaml [hunk 1]
```diff
@@ -70,13 +70,13 @@ objects:
           kind: ImageStreamTag
           name: runtime:latest
         incremental: true
-        env:
+        env:
```
Comment (Ladicek) [inline greeting-service/.openshiftio/application.yaml]: whitespace

File: greeting-service/pom.xml [hunk 2]
```diff
@@ -14,34 +14,34 @@
 
    <parent>
       <groupId>io.openshift.booster</groupId>
-      <artifactId>wfswarm-circuit-breaker-parent</artifactId>
-      <version>7.1.0-redhat-3-SNAPSHOT</version>
+      <artifactId>thorntail-circuit-breaker-parent</artifactId>
+      <version>2.2.0.ER5</version>
```
Comment (Ladicek) [inline greeting-service/pom.xml]: `2.2.0-redhat-1-SNAPSHOT`

File: name-service/pom.xml [hunk 3]
```diff
@@ -14,24 +14,24 @@
 
    <parent>
       <groupId>io.openshift.booster</groupId>
-      <artifactId>wfswarm-circuit-breaker-parent</artifactId>
-      <version>7.1.0-redhat-3-SNAPSHOT</version>
+      <artifactId>thorntail-circuit-breaker-parent</artifactId>
+      <version>2.2.0.ER5</version>
```
Comment (Ladicek) [inline name-service/pom.xml]: `2.2.0-redhat-1-SNAPSHOT`

File: name-service/src/test/java/io/openshift/booster/NameEndpointTest.java [hunk 4]
```diff
@@ -44,59 +44,24 @@
     @Test
     @RunAsClient
     public void testGetName() {
-        when()
-                .get(BASE_URI + "/name")
-        .then()
-                .assertThat()
-                .statusCode(200)
-                .body(equalTo("World"));
+        RestAssured.when().get(BASE_URI + "/name").then().assertThat().statusCode(200).body(equalTo("World"));
```
Comment (Ladicek) [inline name-service/src/test/java/io/openshift/booster/NameEndpointTest.java:47]: Is this how it looks in community?

File: pom.xml [hunk 5]
```diff
@@ -15,24 +15,24 @@
...
       <artifactId>booster-parent</artifactId>
-      <version>20</version>
+      <version>23</version>
    </parent>
 
    <groupId>io.openshift.booster</groupId>
-   <artifactId>wfswarm-circuit-breaker-parent</artifactId>
-   <version>7.1.0-redhat-3-SNAPSHOT</version>
+   <artifactId>thorntail-circuit-breaker-parent</artifactId>
+   <version>2.2.0.ER5</version>
```
Comment (Ladicek) [inline pom.xml]: `2.2.0-redhat-1-SNAPSHOT`

File: pom.xml [hunk 6]
```diff
@@ -44,62 +44,34 @@
...
             <type>pom</type>
             <scope>import</scope>
          </dependency>
 
          <dependency>
-            <groupId>org.wildfly.swarm</groupId>
-            <artifactId>bom-certified</artifactId>
-            <version>${version.wildfly.swarm}</version>
-            <type>pom</type>
-            <scope>import</scope>
```
Comment (Ladicek) [inline pom.xml]: We shouldn't really remove `bom-certified`, that's where we get Arquillian from.

File: name-service/src/test/java/io/openshift/booster/NameEndpointTest.java (hunk 4 above)
Comment (juangon) [reply name-service/src/test/java/io/openshift/booster/NameEndpointTest.java:47]: Yes

File: tests/pom.xml [hunk 7]
```diff
@@ -14,14 +14,14 @@
 
    <parent>
       <groupId>io.openshift.booster</groupId>
-      <artifactId>wfswarm-circuit-breaker-parent</artifactId>
-      <version>7.1.0-redhat-3-SNAPSHOT</version>
+      <artifactId>thorntail-circuit-breaker-parent</artifactId>
+      <version>2.2.0.ER5</version>
```
Comment (Ladicek) [inline tests/pom.xml]: `2.2.0-redhat-1-SNAPSHOT`

File: pom.xml (hunk 6 above)
Comment (juangon) [reply pom.xml]: Oh ok, this what was done in community but really affects prod. Gonna change that

File: tests/pom.xml [hunk 8]
```diff
@@ -136,74 +135,9 @@
...
-                       -
-                       - And here, we add the correct versions of the artifacts to test classpath. Unfortunately,
-                       - this is only possible by file paths, not by artifact coordinates.
-                       -->
-                     <additionalClasspathElements>
-                        <additionalClasspathElement>${project.build.directory}/dependency/jackson-core.jar</additionalClasspathElement>
-                        <additionalClasspathElement>${project.build.directory}/dependency/jackson-annotations.jar</additionalClasspathElement>
-                        <additionalClasspathElement>${project.build.directory}/dependency/jackson-databind.jar</additionalClasspathElement>
-                     </additionalClasspathElements>
-                  </configuration>
```
Comment (Ladicek) [inline tests/pom.xml:206]: Perfect, this removal was long overdue!

File: tests/src/test/java/io/openshift/booster/OpenshiftIT.java [hunk 9]
```diff
@@ -113,11 +114,11 @@ private boolean testCircuitBreakerState(String expectedState) {
     }
 
     private void changeNameServiceState(String state) {
-        given()
+        RestAssured.given()
                 .header("Content-type", "application/json")
                 .body(Json.createObjectBuilder().add("state", state).build().toString())
                 .put(nameServiceUrl + "api/state")
-        .then()
+                .then()
```
Comment (Ladicek) [inline tests/src/test/java/io/openshift/booster/OpenshiftIT.java:121]: Is this how it looks in community?

File: pom.xml (hunk 6 above)
Comment (Ladicek) [reply pom.xml]: Yea community doesn't need `bom-certified` (it actually doesn't even exist in community releases).

File: tests/src/test/java/io/openshift/booster/OpenshiftIT.java (hunk 9 above)
Comment (juangon) [reply tests/src/test/java/io/openshift/booster/OpenshiftIT.java:121]: Yes

**Comment by juangon** [comment]
@Ladicek pushed changes

**Review by Ladicek — APPROVED** [review]

**Pull request merged**